### PR TITLE
[md/mdx] more explicit deprecation and headings reorg

### PR DESCRIPTION
**Pull request by sarah11918**
- New or updated content

This PR
- Adjusts the deprecation notice to be super clear about the **features** being deprecated (not "Markdown")
- Reorganizes some headings to support focus on Pages 
- updates an internal link pointing to an anchor tag that changed (Thank you, Hippo!)

### CODE REVIEW

File: src/pages/en/guides/markdown-content.md [hunk 1]
```diff
@@ -112,11 +117,11 @@ An example blog post `content` object might look like:
...
 :::
 
-#### Layouts for multiple page types
+#### Example: Using one Layout that works for `.md`, `.mdx`, and `.astro` files
 
-An Astro layout can receive both the content object from `.md` and `.mdx` files, as well as any named props passed from `.astro` files.
+A single Astro layout can be written to receive the content object from `.md` and `.mdx` files, as well as any named props passed from `.astro` files.
 
-In the example below, the layout will display the page title either from an Astro component passing a `title` attribute, or from a frontmatter YAML `title` property:
+In the example below, the layout will display the page title either from an Astro component passing a `title` attribute, or from a frontmatter YAML `title` property :
```
Comment (Yan-Thomas) [inline src/pages/en/guides/markdown-content.md:124]: yup, it's me again being picky with commas (and little sneaky whitespaces).
```suggestion
In the example below, the layout will display the page title either from an Astro component passing a `title` attribute or from a frontmatter YAML `title` property:
```

**Review by Yan-Thomas — COMMENTED** [review]
Nice work, thanks for this @sarah11918! I added a small nit suggestion but LGTM.

File: src/pages/en/guides/markdown-content.md [hunk 2]
```diff
@@ -190,35 +195,26 @@ export default defineConfig({
...
 
-## Astro Markdown
+### Variables and Components
 
 :::caution[Deprecated]
-Astro v1.0 Release Candidate (RC) [no longer supports components or JSX in Markdown pages by default](/en/migrate/#deprecated-components-and-jsx-in-markdown) and may be removed in a future release. 
+Astro v1.0 Release Candidate (RC) **only supports standard Markdown in `.md` files**. The ability to use [components or JSX in Markdown pages is no longer enabled by default](/en/migrate/#deprecated-components-and-jsx-in-markdown) and support will eventually be removed entirely. 
 
-In the meantime, Astro config supports a [legacy flag](/en/reference/configuration-reference/#legacyastroflavoredmarkdown) that will re-enable these features in Markdown pages until you are able to migrate to MDX in Astro.
+Astro config supports a [legacy flag](/en/reference/configuration-reference/#legacyastroflavoredmarkdown) that will re-enable these features in Markdown pages until you are able to migrate to MDX in Astro, which is the recommended path forward if you need more features than standard Markdown provides.
```
Comment (sarah11918) [inline src/pages/en/guides/markdown-content.md:203]:
```suggestion
Astro config supports a [legacy flag](/en/reference/configuration-reference/#legacyastroflavoredmarkdown) that will re-enable these features in Markdown pages until you are able to migrate to MDX in Astro. The Astro MDX integration is the recommended path forward if you need more features than standard Markdown provides.
```

File: src/pages/en/guides/markdown-content.md [hunk 3]
```diff
@@ -190,35 +195,26 @@ export default defineConfig({
...
-Astro v1.0 Release Candidate (RC) [no longer supports components or JSX in Markdown pages by default](/en/migrate/#deprecated-components-and-jsx-in-markdown) and may be removed in a future release. 
+Astro v1.0 Release Candidate (RC) **only supports standard Markdown in `.md` files**. The ability to use [components or JSX in Markdown pages is no longer enabled by default](/en/migrate/#deprecated-components-and-jsx-in-markdown) and support will eventually be removed entirely. 
 
-In the meantime, Astro config supports a [legacy flag](/en/reference/configuration-reference/#legacyastroflavoredmarkdown) that will re-enable these features in Markdown pages until you are able to migrate to MDX in Astro.
+Astro config supports a [legacy flag](/en/reference/configuration-reference/#legacyastroflavoredmarkdown) that will re-enable these features in Markdown pages until you are able to migrate to MDX in Astro. The Astro MDX integration is the recommended path forward if you need more features than standard Markdown provides.
 :::
 
-### Using Variables in Markdown
+Please install the official [`@astrojs/mdx`](/en/guides/integrations-guide/mdx/) integration to continue to use:
```
Comment (sarah11918) [inline src/pages/en/guides/markdown-content.md:206]:
```suggestion
Please install the official [`@astrojs/mdx`](/en/guides/integrations-guide/mdx/) integration to use:
```

File: src/pages/en/guides/markdown-content.md [hunk 4]
```diff
@@ -190,35 +195,26 @@ export default defineConfig({
...
-Please install the official [`@astrojs/mdx`](/en/guides/integrations-guide/mdx/) integration to continue to use [Astro components](/en/core-concepts/astro-components/) or [UI framework components](/en/core-concepts/framework-components/) in MDX (`.mdx`) files.
+- [Astro components](/en/core-concepts/astro-components/) or [UI framework components](/en/core-concepts/framework-components/) in MDX (`.mdx`) files.
 
 See the migration guide for help [converting your existing Astro `.md` files to `.mdx`](/en/migrate/#converting-existing-md-files-to-mdx).
 
-
-## MDX
+## MDX Features
 
 Astro includes full support for MDX with the [official `@astrojs/mdx` integration](/en/guides/integrations-guide/mdx/).
```
Comment (sarah11918) [inline src/pages/en/guides/markdown-content.md:216]:
```suggestion
Astro includes full support for MDX with the official `@astrojs/mdx` integration. These support the deprecated features in the previous section and more to enhance your Markdown authoring. See the [MDX integration guide](/en/guides/integrations-guide/mdx/) for more information.
```

File: src/pages/en/guides/markdown-content.md [hunk 5]
```diff
@@ -70,8 +70,13 @@ const { content } = Astro.props;
 </html>
 ```
 
-The `content` prop also contains an `astro` property with additional metadata about the page such as the complete Markdown `source` and a `headers` object.
+### Markdown Props
 
+The `content` prop also contains an `astro` property with additional metadata about a Markdown page such as the complete Markdown `source` and a `headers` object.
+
+:::note
+Markdown and MDX files do not return identical `content` objects. See the MDX integration guide for [MDX properties exposed](/en/guides/integrations-guide/mdx/#exported-properties)
```
Comment (sarah11918) [inline src/pages/en/guides/markdown-content.md:78]:
```suggestion
Markdown and MDX files do not return identical `content` objects. See the MDX integration guide for [MDX properties exposed](/en/guides/integrations-guide/mdx/#exported-properties).
```

File: src/pages/en/guides/markdown-content.md [hunk 6]
```diff
@@ -33,7 +33,7 @@ Markdown does support **bold** and _italics._
 To learn more about adding a layout to your page, read the next section on **Markdown Layouts.**
 ```
 
-### Markdown Layouts
+### Frontmatter `layout:`
```
Comment (hippotastic) [inline src/pages/en/guides/markdown-content.md:36]:
```suggestion
### Frontmatter `layout`
```

I'd probably remove the trailing colon as it's not part of the property name itself, and it looks unusual both in the heading and the sidebar.

File: src/pages/en/guides/markdown-content.md [hunk 7]
```diff
@@ -46,7 +46,7 @@ layout: ../layouts/BaseLayout.astro
 
 A typical layout for Markdown pages includes:
 
-1. the content prop to access the Markdown page's frontmatter and other data.
+1. the content prop to access the Markdown or MDX page's frontmatter and other data.
```
Comment (hippotastic) [inline src/pages/en/guides/markdown-content.md:49]:
```suggestion
1. the `content` prop to access the Markdown or MDX page's frontmatter and other data.
```

**Review by hippotastic — APPROVED** [review]
LGTM! Two nits included, your choice whether to fix them or not :)

**Comment by kylebutts** [comment]
One small thing: the only frontmatter passed here is `title`, but the layout prop right below uses `title`, `author` and `date`. Is it worth adding those props in for continuity's sake? Especially since this section reads like a tutorial *"To start using Markdown in Astro,"*

CleanShot 2022-08-07 at 16 44 50@2x
CleanShot 2022-08-07 at 16 45 36@2x

**Comment by sarah11918** [comment]
Hmm... I see what you're saying, but I guess I hadn't considered the first example related since it doesn't have a `layout:` in its front matter, and it's in a different section. 

I'm shutting down the laptop for the night, but this page is not done yet, so I will absolutely keep this in mind for the next pass! Good note, thanks!

**Pull request merged**